source/UIAUtils.py

```diff
@@ -0,0 +1,121 @@
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 20015-2016 NV Access Limited
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+
+from comtypes import COMError
+import UIAHandler
+
+def createUIAMultiPropertyCondition(*dicts):
+	"""
+	A helper function that Creates a complex UI Automation Condition matching on various UI Automation properties with both 'and' and 'or'.
+	Arguments to this function are dicts whos keys are UI Automation property IDs, and whos values are a list of possible values for the property ID.

```

Comment:
I was experimenting with this. 
I created a class called NotCondition, and if the function detected one of the values of any list item was wrapped in a NotCondition, it would do a not condition query around the innerOr. Is there room to add this to generalize this function more?